schema.graphql

```diff
@@ -1,3 +1,111 @@
+# ADD TOTAL TOKENS
+# spread fees are expresed in sell token
+# write a how to query on a specific block
+
+
+######################################################################################################################
+type Totals @entity {
+  "This is a singleton entity to contain accumulators for all values. Id will be always 1"
+  id: ID!
+  "Total number of tokens traded"
+  tokens: BigInt!
+  "Total number of orders placed"
+  orders: BigInt!
+  "Total number of traders"
+  traders: BigInt!
+  "Total number of batches settled"
+  settlements: BigInt!
+  "Total traded volume in USD"
+  volumeUsd: BigDecimal
+  "Total traded volume in ETH"
+  volumeEth: BigDecimal
+  "Total fees in USD"
+  feesUsd: BigDecimal
+  "Total fees in Eth"
+  feesEth: BigDecimal
+}
+
+######################################################################################################################
+type DailyTotals @entity {
+  "Day timestamp"
+  id: ID!
+  "Start day timestamp"
+  timestamp: BigInt!
+  "Total number of tokens traded"
+  tokens: BigInt!
+  "Total number of orders placed"
+  orders: BigInt!
+  "Total number of batches settled"
+  settlements: BigInt!
+  "Total traded volume in USD"
+  volumeUsd: BigDecimal
+  "Total traded volume in ETH"
+  volumeEth: BigDecimal
+  "Total fees in USD"
+  feesUsd: BigDecimal
+  "Total fees in Eth"
+  feesEth: BigDecimal
+  "Traded tokens"
+  tokens: [Token!]!
+}
+
+######################################################################################################################
+type HourlyTotals @entity {
+  "Hour timestamp"
+  id: ID!
+  "Start hour timestamp"
+  timestamp: BigInt!
+  "Total number of tokens traded"
+  tokens: BigInt!
+  "Total number of orders placed"
+  orders: BigInt!
+  "Total number of batches settled"
+  settlements: BigInt!
+  "Total traded volume in USD"
+  volumeUsd: BigDecimal
+  "Total traded volume in ETH"
+  volumeEth: BigDecimal
+  "Total fees in USD"
+  feesUsd: BigDecimal
+  "Total fees in Eth"
+  feesEth: BigDecimal
+  "Traded tokens"
+  tokens: [Token!]!
+}
+
+######################################################################################################################
+# It makes sense to add symbol, decimals and name? I mean for making a single query to fetch that data
+type TokenDailyTotals @entity {
+  "TokenAddress + timestamp day"
+  id: ID!
+  "Token address"
+  address: Bytes!
+  "Start day timestamp"
+  timestamp: BigInt!
+  "Total amount traded that day in USD"
+  totalVolumeUsd: BigDecimal!
+  "Total amount traded that day in ETH"
+  totalVolumeEth: BigDecimal!
+  "Number of trades that day"
+  totalTrades: BigInt!
```

Comment:
actually here u used the naming convention of `totalTrades`, so nice, it would be coherent with the above